web/src/features/panels/zone/ZoneDetails.tsx

```diff
@@ -58,6 +64,52 @@ export default function ZoneDetails(): JSX.Element {
 
   useScrollHashIntoView(isLoading);
 
+  const datetimes = useMemo(
+    () => Object.keys(data?.zoneStates || {})?.map((key) => new Date(key)),
+    [data]
+  );
```

Comment:
This was recomputed each time, causing re-render of child components depending on it